services/userData.js

```diff
@@ -0,0 +1,78 @@
+const randomUUID = require('uuid');
+const sequelize = require('../models');
+const bcrypt = require('bcrypt');
+
+exports.getAll = async ctx => {
+  return sequelize.User.findAll();
+};
+
+exports.getOne = async username => {
+  const user = await sequelize.User.findOne({
+    where: { username: username },
+  });
+  return user;
+};
+
+exports.createOne = async (username, password, email) => {
+  const id = randomUUID();
+  const salt = await bcrypt.genSalt(10);
+  const pass = await bcrypt.hash(password, salt);
+  const newUser = {
+    username: username,
+    password: pass,
+    salt: salt,
+    email: email,
+    id: id,
+  };
+  try {
+    await sequelize.User.create(newUser);
+    const createdUser = sequelize.User.findByPk(id);
+    return createdUser;
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+exports.login = async (username, password) => {
+  const user = await sequelize.User.findOne({
+    where: {
+      username: username,
+    },
+  });
+  const hashedPassword = user.password;
+  const newHashedPassword = await bcrypt.hash(password, user.salt);
+  if (hashedPassword === newHashedPassword) {
+    return console.log('logged');
+  } else return console.log('false');
+};
+
+exports.deleteOne = async param => {
+  await sequelize.User.destroy({
+    where: { id: param },
+    force: true,
+  });
+  return sequelize.User.findAll();
+};
+
+exports.updateAge = async (username, age) => {
```

Comment:
You don't need to write an update function for every user attribute, make 1 update function, which will be able to update any property, besides come fields, for example, password and email.